src/shared/api/api.js

```diff
@@ -55,8 +55,19 @@ function prefillMethod(method) {
     })
 }
 
-function graphql({ provider, query, variables = {}, signal }) {
-  const uri = `${config.API_URL}/graphql/${provider}`
+function graphql({
+  provider,
+  query,
+  variables = {},
+  signal,
+  supportsServiceless = false,
+}) {
+  let uri = `${config.API_URL}/graphql/${provider}`
+
+  if (supportsServiceless && !provider) {
+    uri = `${config.API_URL}/graphql/`
```

Comment:
Is the `/` needed when the original one doesn't have it?

Reply:
yah the endpoint requires`/` unlike when there are providers passed